Update device API

dbitbox opened the pull request:
Fixes #71, #72

jonasschnelli commented:
Regarding #71 (xpub). I think it would be nice if we could place a sha256 of the master xpub key. This would allow consuming apps to link to a wallet/address cache.

dbitbox commented:
Added a sha(xpub) as a 'device id'. Removed standalone `device:serial` and `device:version` commands as they can be accessed with `device:info`.

Example:
**send**
```
{ "device": "info"}
```
**reply**
```
{ "device": {
    "serial":"01010101010101010101010101010101",
    "version":"v1.2.0-19-g3cf2f77", 
    "name":"Digital Bitbox",
    "id":"daf3de89bce1ead7cd7ca7d22a8f3513e39f28c3b753c4af2a7ba2b9a4a80995", 
    "seeded":true, 
    "lock":false
    }
}
```
